rest-api/tools/import_organizations.py

```diff
@@ -26,7 +30,16 @@
 from model.site_enums import SiteStatus, EnrollingStatus, DigitalSchedulingStatus
 from participant_enums import OrganizationType
 from main_util import get_parser, configure_logging
+from tools.import_participants import _setup_questionnaires, import_participants
 
+
+# Environments
+ENV_TEST = 'pmi-drc-api-test'
+ENV_STAGING = 'all-of-us-rdr-staging'
+ENV_STABLE = 'all-of-us-rdr-stable'
+ENV_PROD = 'all-of-us-rdr-prod'
+ENV_SANDBOX = 'all-of-us-rdr-sandbox'
```

Comment:
I don't think we want to bother with a separate column for sandbox... we could just test for any project other than test, staging, stable, or prod. So you can probably remove this?